src/Orleans/Serialization/BuiltInTypes.cs

```diff
@@ -1850,6 +1852,7 @@ internal static void SerializeInvokeMethodRequest(object obj, ISerializationCont
         internal static object DeserializeInvokeMethodRequest(Type expected, IDeserializationContext context)
         {
             int iid = context.StreamReader.ReadInt();
+            ushort iVersion = context.StreamReader.ReadUShort();
```

Comment:
As I mentioned in another comment, do we care about it? Deploying different versions of Orleans itself side by side is not supported. This PR enables deploying 2 different *app* versions side by side, as long as they both use the same version of Orleans that enables versioning.